src/useKeyboard.ts

```diff
@@ -50,17 +50,17 @@ export function useKeyboard() {
     ]
 
     return () => {
-      if (Keyboard.removeListener) {
+      if (typeof subscriptions?.[0]?.remove === 'function') {
```

Comment:
`subscriptions[0]` always exists here.

```suggestion
      if (typeof subscriptions[0].remove === 'function') {
```

Reply:
I recheck it on latest versions

Reply:
you right  RN [0.59.x...0.6x.x] always return EmitterSubscription with remove method, so this `if` block don't make sense anymore

https://github.com/facebook/react-native/blob/c20070f10458d48d6ac1eaac49e681e932bfb9fd/Libraries/EventEmitter/NativeEventEmitter.js#L41-L50